index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <navbar id="navbar">
+      <a href="#projects">Projects</a>
+      <a href="#">About</a>
+      <a id="profile-link" target="_blank">my github</a>
+    </navbar>
+    <section id="welcome-section">
```

Comment:
> The welcome-section should be a header tag to have better semantics and make it more readable.

- [ ] Good one on figuring this but I would implore you to be polite in your comment, in this case, it can be `I would implore you to... or I think it would be better...` and also it would be more explanatory to the student if you give a reason for your comments.